docs/user/troubleshooting.md

```diff
@@ -66,11 +66,10 @@ Master nodes waiting for Ignition is indicative of problems on the bootstrap nod
 
 If the bootstrap node isn't available, first double check that it hasn't been automatically removed by the installer. If it's not being created in the first place, the installer will need to be [troubleshot](#installer-fails-to-create-resources).
 
-After using SSH to access the bootstrap node, the most important thing to look at is `bootkube.service`. The logs can be viewed with the following command:
+The most important thing to look at on the bootstrap node is `bootkube.service`. The logs can be viewed in two different ways:
 
-```sh
-journalctl --unit=bootkube.service
-```
+1. If SSH is available, the following command can be run on the bootstrap node: `journalctl --unit=bootkube.service`
+2. Regardless of whether or not SSH is available, the following command can be run: `curl --insecure --cert ${INSTALL_DIR}/tls/journal-gatewayd-client.crt --key ${INSTALL_DIR}/tls/journal-gatewayd-client.key 'https://${BOOTSTRAP_IP}:19531/entries?follow&_SYSTEMD_UNIT=bootkube.service'`
```

Comment:
Should the installer connect to this automatically and stream it into the installer logs (like we do for Terraform) instead of [polling for the API][1]?  I'm fine leaving that to follow-up work, but I don't think we need to document this manual approach if we're planning on adding a built-in approach in the future.

[1]: https://github.com/openshift/installer/blob/15937bee9a4c8d3f31f2576f858e4805020a274f/cmd/openshift-install/create.go#L193-L225

Reply:
That's a good question. I'd prefer to always stream these logs, but I'm not sure how to handle the case where a network connection is not available. Do we just optimistically stream them and ignore it otherwise?

Reply:
I once wrote a simple package to talk to gatewayd, in case you ever do want to do that: https://github.com/coreos/go-systemd/pull/86

Also: Having an installer sub-command to that does the equivalent `curl` would be pretty nice.